### PR TITLE
Schema description updates in transactions protocol.

### LINKED ISSUE #170: [docs] "signed_transaction schema" description fix
File: [docs/04_protocol/02_transactions_protocol.md](https://github.com/EOSIO/welcome/blob/develop/docs/04_protocol/02_transactions_protocol.md)

In 'signed_transaction schema' section,
* description of 'expiration' field
time transaction must be confirmed by  ==> the time at which a transaction expires
* description of 'ref_block_num' field
block number in last 216 blocks ==> lower 16 bits of a block num in the last 2^16 blocks
* description of 'ref_block_prefix' field
lower 32 bits of block ID ==> lower 32 bits of the block id referred by 'ref_block_num'

### PR DESCRIPTION
**Pull request by bobgt**
Resolves #170 

Verified the description suggestions given in issue #170 by @taokayan

### CODE REVIEW

File: docs/04_protocol/02_transactions_protocol.md [hunk 1]
```diff
@@ -104,16 +104,16 @@ A signed transaction extends the basic contents of the transaction schema to inc
...
-`max_net_usage_words` | `unsigned_int` | upper limit on total network bandwidth billed (in 64-bit words)
-`max_cpu_usage_ms` | `uint8_t` | upper limit on total CPU time billed (in miliseconds)
-`delay_sec` | `unsigned_int` | number of seconds to delay transaction for
-`context_free_actions` | array of `action` | list of context-free actions if any
-`actions` | array of `action` | list of [action instances](#343-action-instance)
-`transaction_extensions` | `extensions_type` | extends fields to support additional features
-`signatures` | array of `signature_type` | digital signatures after transaction is signed
+`expiration` | `time_point_sec` | the time at which a transaction expires
+`ref_block_num` | `uint16_t` | the lower 16 bits of a block num in the last 2^16 blocks
+`ref_block_prefix` | `uint32_t` | the lower 32 bits of the block id referred by `ref_block_num'
```
Comment (lparisc) [inline docs/04_protocol/02_transactions_protocol.md]: lower 32 bits of the block id --> lower 32 bits of block ID

File: docs/04_protocol/02_transactions_protocol.md [hunk 2]
```diff
@@ -104,16 +104,16 @@ A signed transaction extends the basic contents of the transaction schema to inc
...
-`ref_block_num` | `uint16_t` | block number in last 2<sup>16</sup> blocks
-`ref_block_prefix` | `uint32_t` | lower 32 bits of block ID
-`max_net_usage_words` | `unsigned_int` | upper limit on total network bandwidth billed (in 64-bit words)
-`max_cpu_usage_ms` | `uint8_t` | upper limit on total CPU time billed (in miliseconds)
-`delay_sec` | `unsigned_int` | number of seconds to delay transaction for
-`context_free_actions` | array of `action` | list of context-free actions if any
-`actions` | array of `action` | list of [action instances](#343-action-instance)
-`transaction_extensions` | `extensions_type` | extends fields to support additional features
-`signatures` | array of `signature_type` | digital signatures after transaction is signed
+`expiration` | `time_point_sec` | the time at which a transaction expires
```
[the remark this answers is not in the document]
Comment (bobgt) [reply docs/04_protocol/02_transactions_protocol.md]: > Quite frankly the original description provided more information by clarifying that the transaction must be confirmed before that time. The proposed change just reaffirms the obvious. I suggest: "**time transaction must be confirmed by before it expires**" or something of the sort.

When I read through **time transaction**, it sounds one phrase word to me which is incomplete at the end. How about we add article **the** for this description for more clarity? 

**the time the transaction must be confirmed by before it expires**

Would that make sense?

File: docs/04_protocol/02_transactions_protocol.md [hunk 3]
```diff
@@ -104,16 +104,16 @@ A signed transaction extends the basic contents of the transaction schema to inc
...
+`ref_block_num` | `uint16_t` | the lower 16 bits of a block num in the last 2^16 blocks
```
[the remark this answers is not in the document]
Comment (bobgt) [reply docs/04_protocol/02_transactions_protocol.md]: Screenshot 2020-03-24 at 1 29 06 AM


We already have this syntax is place for the exponent but it is not rendering. See the attached screenshot from the live portal. Any idea why it is not rendering?

File: docs/04_protocol/02_transactions_protocol.md [hunk 4]
```diff
@@ -104,16 +104,16 @@ A signed transaction extends the basic contents of the transaction schema to inc
...
+`ref_block_prefix` | `uint32_t` | the lower 32 bits of the block id referred by `ref_block_num'
+`max_net_usage_words` | `unsigned_int` | the upper limit on total network bandwidth billed (in 64-bit words)
+`max_cpu_usage_ms` | `uint8_t` | the upper limit on total CPU time billed (in miliseconds)
+`delay_sec` | `unsigned_int` | the number of seconds to delay transaction for
+`context_free_actions` | array of `action` | the list of context-free actions if any
+`actions` | array of `action` | the list of [action instances](#343-action-instance)
+`transaction_extensions` | `extensions_type` | the extends fields to support additional features
```
[the remark this answers is not in the document]
Comment (bobgt) [reply docs/04_protocol/02_transactions_protocol.md]: Good point. 

For the first description, please see my comment above.

File: docs/04_protocol/02_transactions_protocol.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lparisc) [reply docs/04_protocol/02_transactions_protocol.md]: Yes, that's fine.

File: docs/04_protocol/02_transactions_protocol.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (lparisc) [reply docs/04_protocol/02_transactions_protocol.md]: This is a bug. We should file an issue here: https://github.com/EOSIO/eosio-docs-gatsby

[the remark this answers is not in the document]
Comment (lparisc) [reply docs/04_protocol/02_transactions_protocol.md]: Done [here](https://github.com/EOSIO/eosio-docs-gatsby/issues/129). In the meantime, try using LaTex and see if it renders properly within the table:

"2<sup>16</sup>" --> "$2^{16}$"

[the remark this answers is not in the document]
Comment (bobgt) [reply docs/04_protocol/02_transactions_protocol.md]: Thanks for filing an issue 

The LaTex sytax worked!

File: docs/04_protocol/02_transactions_protocol.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (bobgt) [reply docs/04_protocol/02_transactions_protocol.md]: Removed **the** from the descriptions except for the first one.

File: docs/04_protocol/02_transactions_protocol.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (dskvr) [reply docs/04_protocol/02_transactions_protocol.md]: It's not a bug.  You need to enclose in latex or use unicode.

**Pull request merged**